### ISSUE TITLE
[zod-openapi] RPC Mode Fails Without Request Parameters

### ISSUE THREAD

**Issue by akineko** [honojs/middleware#518]
hono: 4.3.6
@hono/zod-openapi: 0.11.1

When defining an API without request parameters, the RPC mode does not working.
But, it works when setting path parameters or the params property of the request object.

```ts
const app = new OpenAPIHono()
  .openapi(createRoute({
    method: 'get',
    path: '/users',
    request: {},
    responses: {
      200: {
        description: 'users',
        content: {
          'application/json': {
            schema: z.array(
              z.object({
                id: z.string(),
                name: z.string(),
                age: z.number(),
              })
            ),
          },
        },
      },
    },
  }), (c) => {
    return c.json([{
      id: '1',
      name: 'name',
      age: 20,
    }])
  })

const client = hc<typeof app>("http://localhost:3000");
client. // not work
```

`path: '/users'` -> `path: '/users/{id}'`
or
`request: {}` -> `request: { params: z.object({ id: z.string() }) }`

```ts
const app = new OpenAPIHono()
  .openapi(createRoute({
    method: 'get',
    path: '/users/{id}',
    request: {
      params: z.object({ id: z.string() }),
    },
    responses: {
      200: {
        description: 'users',
        content: {
          'application/json': {
            schema: z.array(
              z.object({
                id: z.string(),
                name: z.string(),
                age: z.number(),
              })
            ),
          },
        },
      },
    },
  }), (c) => {
    return c.json([{
      id: '1',
      name: 'name',
      age: 20,
    }])
  })

const client = hc<typeof app>("http://localhost:3000");
client.users // working
```

**Comment by yusukebe** [comment]
Hi @akineko 

This may be fixed by #519, and I've released the new version. Try it.

**Comment by akineko** [comment]
This issue resolved.
thanks!
